pydantic_ai_slim/pydantic_ai/ext/temporal/__init__.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+from dataclasses import replace
+from typing import Any, Callable
+
+from temporalio.client import ClientConfig, Plugin as ClientPlugin
+from temporalio.contrib.pydantic import pydantic_data_converter
+from temporalio.worker import Plugin as WorkerPlugin, WorkerConfig
+from temporalio.worker.workflow_sandbox import SandboxedWorkflowRunner
+
+from pydantic_ai.agent import Agent
+
+from ._agent import temporalize_agent, untemporalize_agent
+from ._logfire import LogfirePlugin
+from ._run_context import TemporalRunContext
+from ._settings import TemporalSettings
+
+__all__ = [
+    'TemporalSettings',
+    'TemporalRunContext',
+    'PydanticAIPlugin',
+    'LogfirePlugin',
+    'AgentPlugin',
+    'temporalize_agent',
+    'untemporalize_agent',
+]
+
+
+class PydanticAIPlugin(ClientPlugin, WorkerPlugin):
+    """Temporal client and worker plugin for Pydantic AI."""
+
+    def configure_client(self, config: ClientConfig) -> ClientConfig:
+        config['data_converter'] = pydantic_data_converter
```

Comment:
Print a warning if the user or another plugin had already set another data converter